packages/cli/src/execution-lifecycle/__tests__/execution-lifecycle-hooks.test.ts

```diff
@@ -320,15 +349,6 @@ describe('Execution Lifecycle Hooks', () => {
 					);
 				});
 
-				it('should handle errors when updating execution', async () => {
-					const error = new Error('Failed to update execution');
```

Comment:
Now that we don't catch errors like these, what is surfaced to the user? An error msg coming from typeorm?

Reply:
Yes. it's one of those errors that should almost never happen, but when it does, I prefer to render it as it is on the frontend.